src/evict/evict_page.c

```diff
@@ -75,19 +77,37 @@ __wt_page_release_evict(WT_SESSION_IMPL *session, WT_REF *ref)
 	(void)__wt_atomic_addv32(&btree->evict_busy, 1);
 
 	too_big = page->memory_footprint >= btree->splitmempage;
-	if ((ret = __wt_evict(session, ref, false)) == 0) {
-		if (too_big)
+
+	/*
+	 * We want to track the timing of how long the call to evict took. The
+	 * reason for forced evict depends on the size of the page's size. If
```

Comment:
`the size of the page's size`, and generally I think this comment could be made shorter while conveying the necessary information - could you take a stab at that.